pkg/es/config/config.go

```diff
@@ -193,6 +195,9 @@ func (c *Configuration) NewClient(logger *zap.Logger, metricsFactory metrics.Fac
 
 // ApplyDefaults copies settings from source unless its own value is non-zero.
 func (c *Configuration) ApplyDefaults(source *Configuration) {
+	if len(c.RemoteReadClusters) == 0 {
+		c.RemoteReadClusters = source.RemoteReadClusters
```

Comment:
Thanks @albertteoh - I didn't realize this nuance with `strings.Split()` but after looking at the docs, this makes sense. I didn't see a great (easy) way around this since the cli takes in strings, so I updated `options.go` to check the length and only split when necessary (otherwise default to `[]string{}`). I added 2 tests cases around options as well to cover this.

Reply:
Thanks for adding the test cases to cover the empty string. I think there's also the following case: `--es.remote-read-clusters=cluster_one,,cluster_two`, where the second item is empty but I think we can consider this a user error.